### PR TITLE
Migrate e2e tests to nuts-node repo

### PR DESCRIPTION
**Pull request by reinkrul**
Pair it with the source code, so that they always match (add/change e2e test and nuts-node code in 1 commit). No need for a separate repo.

### CODE REVIEW

File: nuts.yaml
Comment (gerardsn) [inline nuts.yaml:1]: file should not be here?

File: .gitignore [hunk 1]
```diff
@@ -28,4 +28,4 @@ data
 /*.db
 
 /nuts-node
-nuts.yaml
+./nuts.yaml
```
Comment (gerardsn) [inline .gitignore:31]: if the goal is to include nuts.yaml in the e2e directory is not better to add `!nuts.yaml` in the e2e gitignore? (not sure if that actually works)

**Review by gerardsn — CHANGES_REQUESTED** [review]
Are there any changes to files in the e2e-test directory? reviewing 15k lines is a bit much if nothing has changed.

---
`make fix-copyright` probably does something here

File: .gitignore (hunk 1 above)
Comment (reinkrul) [reply .gitignore:31]: I'll test it

**Comment by reinkrul** [comment]
> Are there any changes to files in the e2e-test directory? reviewing 15k lines is a bit much if nothing has changed.
> 
> `make fix-copyright` probably does something here

IRMA wasn't actually used, so I removed the files.

File: e2e-tests/README.md [hunk 2]
```diff
@@ -0,0 +1,49 @@
...
+
+And than run again with the values swapped:
+```console
+$ IMAGE_NODE_A=nutsfoundation/nuts-node:v5.0.0 \
+IMAGE_NODE_B=nutsfoundation/nuts-node:v4.3.0 \
+./run-tests.sh
+```
+
+> **Note**  
+> The tests are updated to the latest node version, they might break when you use an older node version.
```
Comment (gerardsn) [inline e2e-tests/README.md:49]: can be removed

File: e2e-tests/README.md [hunk 3]
```diff
@@ -0,0 +1,49 @@
+This repository contains end-to-end tests for the [nuts-node](https://github.com/nuts-foundation/nuts-node).
```
Comment (gerardsn) [inline e2e-tests/README.md:1]: can be removed

Comment (gerardsn) [inline e2e-tests/README.md:1]:
```suggestion
```

File: e2e-tests/README.md (hunk 2 above)
Comment (gerardsn) [inline e2e-tests/README.md:49]:
```suggestion
```

**Review by gerardsn — APPROVED** [review]

**Pull request merged**